FeatherCoin/Feathercoin
# commits from bitcoin/master

docs: Add a note about the source code filename naming convention
https://github.com/bitcoin/bitcoin/pull/13312

qa: Log as utf-8
https://github.com/bitcoin/bitcoin/pull/13440

policy: Treat segwit as always active
https://github.com/bitcoin/bitcoin/pull/13120

Migrate verify-commits script to python, run in travis
https://github.com/bitcoin/bitcoin/pull/13066

Enable double-SHA256-for-64-byte code on 32-bit x86
https://github.com/bitcoin/bitcoin/pull/13393
